### DISCUSSION TITLE
copilot在pycharm上不起作用

### DISCUSSION THREAD

**Discussion by onllyxx** [community/community#14754] [Copilot Conversations]
pycharm的版本是2021.2.1，已在pycharm中正确安装了copilot，也已激活了设备，但写代码时未出现提示，是不是有可能是我网络的问题？ 
![未出现提示](https://user-images.githubusercontent.com/103281302/162888695-0cfb98da-872c-47d3-af9b-4b4c5834311f.png)

**Comment by YBaCuO** [comment]
我也有一样的问题
